environment.yml

```diff
@@ -21,3 +21,5 @@ dependencies:
       - regex>=2023.8.8
       - SPARQLWrapper>=2.0.0
       - tensorflow>=2.11.0
+      - langcodes>=3.0.0
+      - language_data>=1.0.0
```

Comment:
Do these need to get added in the `requirements.txt` too?
Thinking it might be the cause for this error ([logs](https://github.com/scribe-org/Scribe-Data/actions/runs/7548886531/job/20551838441?pr=60#step:8:31)) in the CI runs on Ubuntu runners at least?

Reply:
Oh yes, forgot to add them in ```requirements.txt``, Let me push another commit.